contrib/ctags.sh

```diff
@@ -4,17 +4,16 @@
 # See file LICENSE for terms.
 #
 
-get_base() {
-    while ! [ -e "$PWD/.git" ] && [ "$PWD" != "/" ]; do
-        cd ..
-    done
+base="$(realpath $0)"
+cd "$(dirname $base)"/..
```

Comment:
1. IMO all the " are not needed here
2. Maybe just `cd $(dirname $0)/..`

Reply:
1. ok to remove if we assume no space in them, but I'd rather generally try to protect variable regardless
2. fixed